apps/miner/lib/miner.ex

```diff
@@ -50,9 +61,12 @@ defmodule Miner do
   end
 
   defp merge_block(coinbase, block) do
-  	new_transactions = [coinbase | block.transactions]
-  	txoids = Enum.map(new_transactions, &(&1.id))
+    new_transactions = [coinbase | block.transactions]
+    txoids = Enum.map(new_transactions, & &1.id)
```

Comment:
parenthesis in areas like this to make it obvious that its a shorthand for function declaration